sdk/storage/storage-file-share/samples/javascript/README.md

```diff
@@ -0,0 +1,68 @@
+---
+page_type: sample
+languages:
+- javascript
+products:
+- azure
+- azure-storage
+urlFragment: storage-file-share-javascript
+---
+
+# Azure Storage File Share client library samples for JavaScript
+
+These sample programs show how to use the JavaScript client libraries for Azure Storage File Shares in some common scenarios.
+
+|__File Name__|__Description__|
+|-------------|---------------|
+|[basic.js][basic]|authenticate with the service using an account name & key (or anonymously with a SAS URL), upload a file, list files and directories, and download a file to a string|
```

Comment:
English nit: I feel like the descriptions should be full sentences, starting with a capital letter and ending with a period

Reply:
I originally wrote them as sentences and found that they all began with "This sample" and then had all the current words except in a different tense. So, I decided to change them to make them feel less repetitive when scanning top-to-bottom. If you feel strongly that they should be full sentences, then I can try to write better/more diversified sentences, or I can change them back, but I think it subtracts from the information in the table.